app/src/features/rules/screens/rulesList/components/RulesList/components/RulesTable/rulesTable.css

```diff
@@ -104,26 +104,43 @@
   font-size: var(--requestly-font-size-sm);
 }
 
+.rule-name-container .rule-name .ant-typography {
+  /* TODO: use theme variables */
+  color: #3ba0ff;
+  transition: color 0.2s ease;
+}
+
+.rule-name-container .rule-name .ant-typography:hover {
+  /* TODO: use theme variables */
+  color: #1f50b1;
+}
```

Comment:
Why we didn't used variables?

Reply:
The rule names in rules table are links whose styles are being applied from top level, and these colors are not present in our tokens.